renderapi/transform/utils.py

```diff
@@ -0,0 +1,90 @@
+from collections import Iterable
+from renderapi.errors import RenderError
+from .leaf import AffineModel, Polynomial2DTransform
+from .transform import TransformList, ReferenceTransform
+
+

```

Comment:
this is where the __all__ = ['estimate_dstpts', 'estimate_transformsum'] should be (and by analogy everywhere else that you want the ability to do an import *